node/node.go

```diff
@@ -128,6 +137,41 @@ func New(conf *Config) (*Node, error) {
 	}, nil
 }
 
+func makeAccountManager(conf *Config) (*accounts.Manager, error) {
```

Comment:
Could you please move this method to `node/config.go` and make sure it meshes nicely with the other methods there? All these configuration dependent subtleties are done there.